image_width: 640
image_height: 480
camera_name: arena_camera
camera_matrix:
  rows: 3
  cols: 3
  data: [362.46905708, 0.0, 293.67520602, 0.0, 362.21510212, 270.5792153, 0.0, 0.0, 1.0]
distortion_model: "fisheye"
distortion_coefficients:
  rows: 1
  cols: 4
  data: [-0.02910789, -0.05947373, 0.086073, -0.04571382]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0]
projection_matrix:
  rows: 3
  cols: 4
  data: [362.46905708, 0.0, 293.67520602, 0.0, 0.0, 362.21510212, 270.5792153, 0.0, 0.0, 0.0, 1.0, 0.0]